### ISSUE TITLE
a11y alternatives for complex diagrams

### ISSUE THREAD

**Issue by cycomachead** [bjc-edc-2017-18/bjc-r#376]
Some images, but not exhaustive:
bjc-r/img/6-computers/logic-gates-quiz-b.png
bjc-r/img/4-internet/redundant-2.jpg

These are images where, in theory, we should have SVGs or something that reveals the text and structure of the image.
There's probably more to do to make them actually usable with a screen reader.

**Comment by cycomachead** [comment]
OK, @maryfries @brianharvey -- when you have a moment can you please check something out. 
This one isn't urgent, but I would say this is the type of accessibility work beyond just legal compliance of alt text we'd want to consider for a proposal / future projects. 

PR's #384 (branch `cycomachead/44-convert-images-to-svg/3`) and #385 (branch `cycomachead/44-convert-images-to-svg/2`)

#384 you'll want to play with the self check (just use the summary) for Unit 4's network diagram questions. 
Without really prompting it all (! -- which is very interesting to me!) Claude built the functionality to dynamically turn each node in the network image on or off. So, it allows you to really easily see network changes w/o having to mentally track which lines you're removing. 
 
-- incidentally, these are also the types of things that'd make great printed worksheets where students could trace paths and such, but that's a different task. :) 

Anyway, should we keep it? It's a good bit of code we'd almost never have written on our own, but it's certainly a useful demo and exploration. The code is thankfully pretty separate, so if we have to turn it off / don't want to maintain it, there's no real downside. 

#385 is less involved, just highlighting nodes -- but still probably useful than just looking at the picture. 

For Unit 6: The AI highlighted that at least AND, OR and NOT were not translated in the Spanish images of the logic gate diagrams. I'm not actually sure what we'd want here.

In both cases, there's definitely some new translations for UI elements and the descriptions/alt text of these images we'd want to verify.

**Comment by brianharvey** [comment]
Sure why not?  Because we're against using AI products in general?

Note that there's an extra connector in the first one, from 6 directly to 10 rather than via 8.
